python_build_requirements.txt

```diff
@@ -1,4 +1,4 @@
 contextlib2==21.6.0
```

Comment:
I renamed this because `requirements` is more standard and I keep forgetting the name. As long as requirements.txt is in there, I'm happy with it.